documentation/source/platform_support.rst

```diff
@@ -0,0 +1,108 @@
+.. _platform-support:
+
+****************
+Platform Support
+****************
+
+cocotb supports Windows, Linux, and macOS, and a wide range of Python versions.
+This page describes which platform combinations we test and support.
+
+The :ref:`platform-support-policy` discusses the underlying policy.
+
+.. note::
+
+  In many cases cocotb will work on platforms not listed on this page.
+  However, we encourage less experienced users to stick with one of the supported options for a good user experience.
+
+  Please reach out to the cocotb team if you would like to see support for a different platform, and are willing to contribute to maintaining it.
+
+Supported Python Versions
+=========================
+
+The following versions of Python (CPython), and all associated patch releases (e.g. 3.8.4), are supported by cocotb.
+
+* Python 3.6
+* Python 3.7
+* Python 3.8
+* Python 3.9
+* Python 3.10
+
+Supported Linux Distributions and Versions
+==========================================
+
+Linux distributions typically ship a default version of Python called "system Python".
+This version of Python can be used with cocotb unless noted otherwise.
```

Comment:
I didn't mention manylinux1 to avoid going in circles. Choosing manylinux1 (for Python 3.6) is a result of us supporting e.g. RHEL7 and Ubuntu 18.04 with system Python. If we stop supporting the last platform that requires manylinux1, then we can freely use another version of manylinux.